### DISCUSSION TITLE
希望能去掉动态页面的话题版块

### DISCUSSION THREAD

**Discussion by tempestte** [the1812/Bilibili-Evolved#1676] [功能建议 / Ideas]
![QQ截图20210324212738](https://user-images.githubusercontent.com/81308753/112318607-30905a00-8ce8-11eb-8c54-58261ed7168b.png)

**Comment by Geeyun-JY3** [comment]
打开 `动态过滤器` 功能后，在「设置面板 > 侧边栏」找到「话题」：
![image](https://user-images.githubusercontent.com/49396421/112337040-8a991b80-8cf8-11eb-9678-2876333679be.png)
